eureka-protempa-etl/pom.xml

```diff
@@ -151,6 +151,12 @@
 							<goal>update</goal>
 						</goals>
 						<configuration>
+							<expressionVars>
+							<property>
+								<name>superuser.username</name>
```

Comment:
Let's namespace the new properties, like eureka.superuser.username. Otherwise, this looks ready to go.